src/common/utils/airport_codes.json

```diff
@@ -46253,7 +46253,13 @@
     "id": "EGCA",
     "id2": "EGCA",
     "british": true,
-    "label": "COAL ASTON - BENTLEY FARM (United Kingdom) (EGCA)"
+    "label": "Coal Aston Airfield / Apperknowle Airstrip (United Kingdom) (EGCA)"
+  },
+  {
+    "id": "COAL ASTON - BENTLEY FARM",
+    "id2": "COAL ASTON - BENTLEY FARM",
+    "british": true,
+    "label": "COAL ASTON - BENTLEY FARM (United Kingdom)"
```

Comment:
Not sure about the significance of using `/`, Do we need that here as well?

Reply:
Sonali did say she thought the `/` played a syntactic role, but, looking at the rest of the file, it doesn't seem to be used consistently  (I left Bentley Farm formatted as I found it)

for e.g.

```json 
 {
    "id": "EGBR",
    "id2": "EGBR",
    "british": true,
    "label": "BREIGHTON - SELBY (United Kingdom) (EGBR)"
  },
```

I could change it to `COAL ASTON / BENTLEY FARM (United Kingdom)` but it's not clear to me if it's right/needed, and it begs the question of whether to do this throughout the file? I am trying to get an explanation from CBP.

I think where the `/` might be significant is in the brackets with the code(s):
```json 
{
    "id": "KJK",
    "id2": "EBKT",
    "british": false,
    "label": "Wevelgem Airport (Belgium) (EBKT / KJK)"
 },
```

hoping CBP can clear this up for me.